packages/devtools_app/test/association_variable_test.dart

```diff
@@ -88,7 +86,7 @@ void main() {
       ),
       isolateRef,
     );
-    when(manager.service.getObject(any, any, offset: 0, count: 2))
+    when(manager!.service!.getObject('433', '123', offset: 0, count: 2))
```

Comment:
can you comment on whether '433' and '123' match some test data somewhere or if they are just arbitrary?

Reply:
see comment above